packages/core/data-transfer/types/remote/protocol/client/index.d.ts

```diff
@@ -4,5 +4,6 @@ import type { TransferMessage } from './transfer';
 export * from './commands';
 export * from './transfer';
 
-export type Message = { uuid: string } & (CommandMessage | TransferMessage);
+export type MessageBody = { uuid: string };
+export type Message = CommandMessage | TransferMessage;
```

Comment:
I wasn't sure if it was constructed as `{ uuid: string } & (CommandMessage | TransferMessage);` for a reason I was missing, but to me it would make more sense to rename MessageBody to Message, CommandMessage and TransferMessage intersection with Message, and then rename this union (the old Message) as something like ValidMessage